execution_engine_testing/test_support/src/wasm_test_builder.rs

```diff
@@ -104,6 +107,8 @@ pub struct WasmTestBuilder<S> {
     standard_payment_hash: Option<ContractHash>,
     /// Auction contract key
     auction_contract_hash: Option<ContractHash>,
+    /// Current "timestamp" for use with the auction.
+    timestamp: u64,
```

Comment:
It looks like this is used only on the caller's side to get the next era end timestamp. It can be tracked outside of the `WasmTestBuilder` as a mutable variable in the context of a unit test.

Reply:
Done